types/chessboardjs/tslint.json

```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
This file should only `"extends": "dtslint/dt.json"`

Reply:
Hi @plantain-00, thanks for the review.